README.md

```diff
@@ -64,14 +64,23 @@ kubectl get crd podchaos.pingcap.com
 * Install Chaos Mesh with Chaos Operator only
 
 ```bash
+# create namespace chaos-testing 
+kubectl create ns chaos-testing
```

Comment:
Did we need to create ns first?

Reply:
Yes, according to https://helm.sh/docs/faq/#automatically-creating-namespaces, Helm 3 returns an error if the namespace does not exist.

Reply:
could you modify the Helm version limitation on `Prerequisites` part? @endurancetest20380119

Reply:
if using helm2, if the namespace exists, did the create command will fail

Reply:
No, Helm 2 will create namespaces automatically if the specified ns doesn't exist.

Reply:
LGTM

Reply:
https://github.com/helm/helm/issues/5753
I'm sorry in words less than.
I'm not very convinced of this fix.

Reply:
@mahjonp @zhouqiang-cl 
I fixed the `Prerequisites` part.